examples/storage/couchdb/README.md

```diff
@@ -0,0 +1,84 @@
+
+## Couchdb 2.0.0 on Kubernetes
+This document describes the steps to create a Couchdb 2.0.0 cluster on Kubernetes.
+
+### Prerequisites
+Access to a kubernetes cluster running v1.4 or later, kubectl installed and
+configured.  Minikube was used for this example. Note that the service type
```

Comment:
Can you write up a non-minikube example as an additional section? 

section one is:
 - here is an example play deployment, where we use minikube
 - let's poke at it
 - let's clean it up

section two:
 - for a prod environment, here is what changes, etc.